x/treasury/internal/keeper/keeper.go

```diff
@@ -67,34 +67,33 @@ func (k Keeper) Send(ctx sdk.Context, fromDid did.Did, toDidOrAddr string, amoun
 }
 
 func (k Keeper) OracleTransfer(ctx sdk.Context, fromDid did.Did,
-	toDidOrAddr string, oracleDid did.Did, amount sdk.Coins) sdk.Error {
+	toDidOrAddr string, oracleDid did.Did, amount sdk.Coins) error {
 
 	// Check if oracle exists
 	if !k.oraclesKeeper.OracleExists(ctx, oracleDid) {
-		return sdk.ErrInternal("oracle specified is not a registered oracle")
+		return sdkerrors.Wrap(types.ErrInternal, "oracle specified is not a registered oracle")
```

Comment:
Just a question; sdk.ErrInternal no longer exists? If it exists ideally we use it instead of having a custom "types.ErrInternal"

Reply:
No, that's why I have created the message type